hypervisor/dm/vpci/vpci_bridge.c

```diff
@@ -74,12 +106,48 @@ static void init_vpci_bridge(struct pci_vdev *vdev)
 	vdev->user = vdev;
 }
 
+/**
+ * @brief Deinitializes the vPCI bridge.
+ *
+ * This function deinitializes the specified virtual PCI device that was previously initialized as a PCI bridge.
+ *
+ * It just set the parent_user and user to NULL.
```

Comment:
For the specified `vdev`, it sets the fields `parent_user` and `user` to NULL, indicating that this virtual device is not owned by any VM.

Since we mentioned such ownership logic in deinit_vpci_bridge(), we should also clarify the ownership in init_vpci_bridge() as these two functions are complementary.

Reply:
> For the specified `vdev`, it sets the fields `parent_user` and `user` to NULL, indicating that this virtual device is not owned by any VM.
> 
> Since we mentioned such ownership logic in deinit_vpci_bridge(), we should also clarify the ownership in init_vpci_bridge() as these two functions are complementary.

Looks like init_vpci_bridge() is not updated yet to clarify the ownership logic?